### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by denisidoro** [chr15m/flk#7]
Hi! Nice project!

Some suggestions, though (maybe I'll work on some of them sometime):

- add tests + CI pipeline 
- stop versioning the flk file and use github releases instead 
- split flk into a sourceable file + bin entrypoint: I would like to reuse flk's data structure implementations in my existing bash scripts, for example, but flk has code that is run outside functions (autorun when sourced) 
- add an flk --eval option 
- add formula to brew

WDYT?

**Comment by chr15m** [comment]
Good ideas, thank you.

 * It is based on [mal](https://github.com/kanaka/mal) which has extensive tests already, but I should probably add tests for the extra functions defined in the project. Added #11 - PRs welcome.
 * Using a release is a good idea, thank you. I added #9 for this.
 * With regards to sourceable file, I would like the binary to be fully self contained, but a sourceable version as a separate artifact is good. #10 is for this. PRs welcome. Did you see that it's possible to "compile" a self contained script from your own Fleck source script?
 * `--eval` sounds good, PRs welcome.
 * I do not use OSX but I'd accept a PR to support the brew thing.
